etl/steps/data/garden/cancer/2024-10-13/gco_cervical_cancer.py

```diff
@@ -0,0 +1,40 @@
+"""Load a meadow dataset and create a garden dataset."""
+
+import owid.catalog.processing as pr
+
+from etl.data_helpers import geo
+from etl.helpers import PathFinder, create_dataset
+
+# Get paths and naming conventions for current step.
+paths = PathFinder(__file__)
+
+
+def run(dest_dir: str) -> None:
+    #
+    # Load inputs.
+    #
+    # Load meadow datasets for Cancer Today and Cancer Over Time datasets.
+    ds_meadow_over_time = paths.load_dataset("gco_cancer_over_time_cervical")
+    ds_meadow_today = paths.load_dataset("gco_cancer_today_cervical")
+
+    # Read tables from meadow datasets.
+    tb_today = ds_meadow_over_time["gco_cancer_over_time_cervical"].reset_index()
```

Comment:
Tiny nit but these are the wrong way round"

tb_over_time = ds_meadow_over_time["gco_cancer_over_time_cervical"].reset_index()
tb_today = ds_meadow_today["gco_cancer_today_cervical"].reset_index()